apps/remix-ide-e2e/src/tests/specialFunctions.test.ts

```diff
@@ -101,7 +101,7 @@ module.exports = {
             .journalLastChildIncludes('to: CheckSpecials.(receive)')
             .journalLastChildIncludes('value: 1 wei')
             .journalLastChildIncludes('data: 0x')
-            .perform(done)
+            .perform(done())
```

Comment:
shouldn't this be `.perform(done)`
or
```
.perform(() => { done() })
```

Reply:
https://nightwatchjs.org/api/perform.html
It's a function passed nothing unless you want to pass and err
    performAsync(function(err) {
      if (err) {
        done(err);
        return;
      }
      // ...
    });